Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Support.hs

```diff
@@ -23,6 +23,7 @@ module Domain.Action.UI.Support
 where
 
 import Data.OpenApi (ToSchema)
+import qualified Domain.Action.UI.Profile as DP
```

Comment:
Instead of importing this there. Move the imported function inside SharedLogic or Utils and use both side.